plugins/inputs/ceph/ceph.go

```diff
@@ -411,26 +432,53 @@ func decodeStatus(acc telegraf.Accumulator, input string) error {
 	return nil
 }
 
+// decodeStatusFsmap decodes the FS map portion of the output of 'ceph -s'
+func decodeStatusFsmap(acc telegraf.Accumulator, data *CephStatus) error {
+	fields := map[string]interface{}{
+		"up":         data.FSMap.NumUp,
+		"in":         data.FSMap.NumIn,
+		"max":        data.FSMap.NumMax,
+		"up_standby": data.FSMap.NumUpStandby,
+	}
+	acc.AddFields("ceph_fsmap", fields, map[string]string{})
+	return nil
+}
+
 // decodeStatusHealth decodes the health portion of the output of 'ceph status'
```

Comment:
Is this the same as `ceph -s`? The other status decoders seem to mention only that command..

Reply:
The `ceph -s` command output has a number of different sections, and they were each broken down into their own decoders.